### DISCUSSION TITLE
Zoom or crop area

### DISCUSSION THREAD

**Discussion by adolfocartagena** [easytarget/esp32-cam-webserver#246] [Ideas]
Hello everybody!

Would it be possible to crop the image to a certain area? I have a camera in the peephole and only 60% of the image is visible, the rest is black. Thank you
![20220412_224025-00](https://user-images.githubusercontent.com/103574419/163168406-63f03cef-6fee-4bda-bfba-b73240a5cafe.jpg)

**Comment by easytarget** [comment]
I'll leave this here in the discussions, so it is visible, It's a good idea/request, see issue #247 for tracking of it.
